integration-tests/src/test/java/oracle/weblogic/domain/ClusterStatus.java

```diff
@@ -186,6 +245,8 @@ public boolean equals(Object other) {
         .append(maximumReplicas, rhs.maximumReplicas)
         .append(minimumReplicas, rhs.minimumReplicas)
         .append(replicasGoal, rhs.replicasGoal)
+        .append(observedGeneration, rhs.observedGeneration)
+        .append(conditions, rhs.conditions)
```

Comment:
We're sorting the conditions for ClusterStatus (also check DomainStatus):

.append(DomainResource.sortList(conditions), DomainResource.sortList(rhs.conditions))

Reply:
done